src/app/home/tabs/profile/profile.component.html

```diff
@@ -1,91 +1,91 @@
-                <p style="margin-bottom: 0.5rem">{{ '@' + profile.username }}</p>
-                <ion-button fill="solid" size="small" (click)="updateProfileView.set(true)">
-                  Edit Profile
-                  <ion-icon slot="end" name="create-outline"></ion-icon>
+              <div style="display: flex; align-items: flex-start">
+                <ion-button fill="clear" (click)="logout()" style="margin: 10px">
+                  <ion-icon slot="icon-only" name="log-out-outline"></ion-icon>
                 </ion-button>
-              </ion-label>
-            </div>
-            <div style="display: flex; align-items: flex-start">
-              <ion-button fill="clear" (click)="logout()" style="margin: 10px">
-                <ion-icon slot="icon-only" name="log-out-outline"></ion-icon>
-              </ion-button>
+              </div>
             </div>
           </div>
         </div>
-      </div>
-      <div class="ion-padding ion-margin-horizontal">
-        <ion-label>
-          <div class="ion-text-start">{{ profile.bio }}</div>
-        </ion-label>
-      </div>
-      <div class="ion-padding-horizontal ion-padding-top ion-margin-horizontal center-container"
-           style="display: flex; align-items: flex-end">
-        <div class="ion-padding-horizontal">
-          <ion-label class="ion-text-center">
-            <ion-chip color="warning">
-              <div style="display: flex; align-items: center">
-                <ion-icon class="ion-padding-end" name="trophy-outline"></ion-icon>
-                <h2>{{ profile.averageScore || 0 | number:'1.0-0' }}</h2>
-              </div>
-            </ion-chip>
-            <p>Score</p>
+        <div class="ion-padding ion-margin-horizontal">
+          <ion-label>
+            <div class="ion-text-start">{{ profile.bio }}</div>
           </ion-label>
         </div>
-        @if (connections$ | async; as connections) {
+        <div class="ion-padding-horizontal ion-padding-top ion-margin-horizontal center-container"
+             style="display: flex; align-items: flex-end">
+          <div class="ion-padding-horizontal">
+            <ion-label class="ion-text-center">
+              <ion-chip color="warning">
+                <div style="display: flex; align-items: center">
+                  <ion-icon class="ion-padding-end" name="trophy-outline"></ion-icon>
+                  <h2>{{ profile.averageScore || 0 | number:'1.0-0' }}</h2>
+                </div>
+              </ion-chip>
+              <p>Score</p>
+            </ion-label>
+          </div>
           <div class="center-container">
             <div class="ion-padding-horizontal">
-              <ion-label class="ion-text-center">
+              <ion-label (click)="openConnections('Followers', connections.followers)" class="ion-text-center">
                 <h1>
                   {{ connections.followers.length }}
                 </h1>
                 <p>followers</p>
```

Comment:
Maybe you could a pointer on hover, else it feels a little bit weird in the browser

Reply:
1c9d4c09750662b225a7e829667fc64599e3ae4a